src/test/java/uk/gov/justice/digital/delius/service/AppointmentServiceTest.java

```diff
@@ -80,6 +82,25 @@ public void appointmentsSortedByContactDateDescending() {
 
     @Test
     public void createsAppointment() {
+        // Given
+        var startTime = OffsetDateTime.now().truncatedTo(ChronoUnit.SECONDS);
+        var endTime = startTime.plusHours(1);
+
+        NewContact deliusNewContactRequest = aDeliusNewContactRequest(startTime, endTime);
```

Comment:
very minor, but prefer `final var`  rather than types to cut down on code noise

Reply:
Will do